### ISSUE TITLE
Midna Conversation Still Causing Hang - Twilight Princess (GC) 

### ISSUE THREAD

**Issue by SamHadavi** [RenolY2/pikminBMG#2]
Despite what's been stated in the release notes for v0.7.3, modifying a BMG file will still cause a hang before conversation with Midna starts.

**Comment by RenolY2** [comment]
Can you test if the issue still persists with this version? https://github.com/RenolY2/pikminBMG/releases/tag/v0.7.4
In my testing talking to Midna no longer seems to lock the game up.
![image](https://github.com/RenolY2/pikminBMG/assets/5006358/ecc7a692-e195-4b11-b7f1-fa8056dd14ec)

**Comment by SamHadavi** [comment]
I unfortunately does not fix it. I should have specified, the only thing I changed was the wallet amounts to the HD version:

300 -> 500
600 -> 1,000
1,000 -> 2,000

I attached a zip of the arc file containing the bmg I edited. Ive been testing the hang with a save that starts at the beginning of the Forest Temple immediately after rescuing the first monkey, I included the gci file for the save in the zip.

[arcandgci.zip](https://github.com/RenolY2/pikminBMG/files/11777553/arcandgci.zip)

**Comment by SamHadavi** [comment]
Okay so I just noticed that ALL in-game conversations cause hang with the edited BMG file. Cutscenes are not affected, but talking to NPCs, prompted or automatic and including Midna by pressing Z, all cause hang.

**Comment by SamHadavi** [comment]
Unfortunately it is still not fixed. I've noticed that ALL in-game conversations cause hang, that includes prompted and automatic as well as Midna by pressing Z. I attached a zip containing the arc file which has the edited bmg inside it. The only things I actually changed was the displayed Rupee amounts for the wallets, they are changed to the amounts in the HD version of Twilight Princess.
[arc.zip](https://github.com/RenolY2/pikminBMG/files/11777663/arc.zip)
